bql/semantic/expression.go

```diff
@@ -136,11 +143,15 @@ func (e *comparisonForLiteral) Evaluate(r table.Row) (bool, error) {
 		csEL, csER string
 	)
 
-	csEL = formatCell(leftCell)
+	csEL, err = formatCell(leftCell)
+	if err != nil {
+		return false, err
+	}
 	csER, err = csLit(e.rS)
 	if err != nil {
 		return false, err
```

Comment:
Here and in line 148 and line 139 I think we should annotate the error expanding on its message to explicitly tell that the error was obtained while evaluating a comparison of a binding and a literal

Reply:
Totally agree. Done.